src/Beman/Optional26/CMakeLists.txt

```diff
@@ -3,24 +3,22 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # cmake-format: on
 
-set(TARGET_LIBRARY "beman_optional26")
+add_library(beman_optional26 STATIC "")
 
-add_library("${TARGET_LIBRARY}" STATIC "")
-
-target_sources("${TARGET_LIBRARY}" PRIVATE optional.cpp detail/iterator.cpp)
+target_sources(beman_optional26 PRIVATE optional.cpp detail/iterator.cpp)
```

Comment:
Per https://github.com/beman-project/Optional26/pull/29#discussion_r1678271229, I think lines 6 and 8 can be merged.